javascript exercise/exercise-3.js

```diff
@@ -0,0 +1,13 @@
+/* 3. Write a JavaScript function to truncate a string to a certain number of words.
+Input: (string, number)
+Output: new string
+Ex: ('The quick brown fox jumps over the lazy dog', 4) => "The quick brown fox" */
+
+function truncateStr(str, number) {
+  let output = "";
+  for (let i = 0; i < number; i++) {
+    output += str.split(" ")[i] + " ";
+  }
+
+  return output.trim();
+}
```

Comment:
cái này kết quả không sai nha nhưng mình có thể `refactor` lại bằng cách:
- đưa `str.split(" ")` ra ngoài `for`, hiện tại cứ mỗi lần loop là lại split
- Hoặc là sử dụng kết hợp `split` `slice` `join`